modules/virt-autoupdate-custom-bootsource.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-autoupdate-custom-bootsource_{context}"]
+= Enabling automatic updates on custom boot sources
+
+{VirtProductName} automatically updates pre-defined boot sources by default. You must manually enable automatic imports and updates on any custom boot sources.
+
+.Procedure
+
+. Use the following command to enable automatic updates on a custom boot source:
++
+[source,terminal]
+----
+$ oc apply -n openshift-cnv -f add-custom-boot-sources.yaml
+----
+
++
+.Example in CentOS 7
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  dataImportCronTemplates:
+  - metadata:
+      name: centos7-image-cron
+      annotations:
+        cdi.kubevirt.io/storage.bind.immediate.requested: "true" (1)
```

Comment:
The `(1)` is readable, but might confuse on copy and paste. Is there a technical better way of showing the 1 in a circle here?

Reply:
Incorrect formatting on my part. Will fix.